android/app/src/main/java/com/rjnr/pocketnode/data/gateway/GatewayRepository.kt

```
@@ -985,8 +1090,163 @@ class GatewayRepository @Inject constructor(
+            // Synthesize predicted change-output cells from in-flight broadcasts.
+            // Without this, rapid sequential sends exhaust live cells before the
+            // light client has synced the change outputs of prior sends — the
+            // observed "Not enough funds available" failure mode.
+            // We include each output of every active pending tx whose lock script
+            // matches the sender's lock (= change output going back to us).
+            // If a pending tx ultimately FAILs, downstream txs that consumed its
+            // synthetic change will also fail and the watchdog times them out.
+            val pendingChange: List<Cell> = pending.flatMap { row ->
+                val pendingTx = try {
+                    json.decodeFromString<Transaction>(row.signedTxJson)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)